### DISCUSSION TITLE
source > Utilization is too high (85)%, pausing for 2 minutes

### DISCUSSION THREAD

**Discussion by anil-inito** [airbytehq/airbyte#32341] [Connector Questions]
I am trying to fetch data from facebook marketing API and selected multiple streams for syncing. I have following questions:

1. Why it is pausing again and again even after giving API advanced access as mentioned in doc
2. If I started 4 table to sync, is there any order in which Airbyte will sync the streams? So if I stop my sync now in between, which stream I need to run again?

Thanks in advance!

**Comment by marcosmarxm** [comment]
1. what is `pausing` in your case? The API is returning 429 error?
2. you can check the sync logs to understand what order was executed to retriever records from your connection. Not all sources/destinations allow to checkpoint and return a failed sync from an intermediary step.

[a comment by an automated account is not shown]
